test/use-swr-key.test.tsx

```diff
@@ -35,11 +35,11 @@ describe('useSWR - key', () => {
     await screen.findByText('data:short request')
 
     await act(() => sleep(100)) // wait 100ms until "long request" finishes
-    screen.getByText('data:short request') // should be "short request" still
+    screen.getByText('data:short request') // should be a "short request" still
```

Comment:
"short request" is the content, not "a short request", same as below
```suggestion
    screen.getByText('data:short request') // should be "short request" still
```